proxyapi/src/output.rs

```diff
@@ -91,6 +92,7 @@ impl HttpHandler for Output {
 
 #[derive(Clone, Debug)]
 pub struct OutputRequest {
+    http_method: Method,
     method: String,
```

Comment:
Here, I wasn't sure why the items were converted into Strings. My guess is to stop heap allocations each frame or a similar performance concern, but if not, then maybe the `http_method` field is an adequate replacement for the existing `method`field.

Reply:
A rude truth: string conversions sucks and it is bad practice in this case but it was my first approach to gui and as the first versions was comfortable. One fo the next steps is to **avoid the String conversion on everything**. For now thank you very much for your effort, i review the code today.